spec/client_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'spec_helper'
+
+describe 'rdiff-backup::client' do
+  [CENTOS_6_OPTS, CENTOS_7_OPTS].each do |pltfrm|
+    context "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(pltfrm).converge(described_recipe)
```

Comment:
There's no good way to test this. I would just ignore it and if anyone has a problem, ask them for a good way to do it.